### ISSUE TITLE
Can´t use: [:escaped_params] ?

### ISSUE THREAD

**Issue by baelter** [sinatra/sinatra#1615]
Shouldn't `:escaped_params` be added here: https://github.com/sinatra/sinatra/blob/master/rack-protection/lib/rack/protection.rb#L33 ??

**Comment by dentarg** [comment]
Seems like it is intentionally not included by `use Rack::Protection`, if that's what you mean: https://github.com/sinatra/sinatra/blob/v2.0.8.1/rack-protection/README.md#cross-site-request-forgery

A bit hard to follow, but I think background for that exist in 

* https://github.com/sinatra/sinatra/issues/310
* https://github.com/sinatra/sinatra/pull/361
* https://github.com/sinatra/rack-protection/issues/6

**Comment by baelter** [comment]
I'm down with that, but it can't be activated with
```ruby
set(:protection, use: [:escaped_params], escape: [:html]
```
Bonus quirk: the `escape` options passed above are used if I activate it with 
```ruby
use Rack::Protection::EscapedParams
```

**Comment by jkowens** [comment]
@baelter I think you are correct, it should be added there and off by default. Can you send up a PR to do that?
